### ISSUE TITLE
Ignis.Components.Web error finding `staticwebassets` directory

### ISSUE THREAD

**Issue by jpsullivan** [DavidVollmers/Ignis#2]
Hi! First off, thanks for putting together this library! I'm in the process of getting it integrated into an application, but I'm unfortunately running into the following error on startup:

```
System.IO.DirectoryNotFoundException: C:\Users\{user}\.nuget\packages\ignis.components.web\0.6.0-preview\staticwebassets\
   at Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(String root, ExclusionFilters filters)
   at Microsoft.AspNetCore.Hosting.StaticWebAssets.StaticWebAssetsLoader.<>c.<UseStaticWebAssetsCore>b__1_0(String contentRoot)
   at Microsoft.AspNetCore.StaticWebAssets.ManifestStaticWebAssetFileProvider..ctor(StaticWebAssetManifest manifest, Func`2 fileProviderFactory)
   at Microsoft.AspNetCore.Hosting.StaticWebAssets.StaticWebAssetsLoader.UseStaticWebAssetsCore(IWebHostEnvironment environment, Stream manifest)
   at Microsoft.AspNetCore.Hosting.StaticWebAssets.StaticWebAssetsLoader.UseStaticWebAssets(IWebHostEnvironment environment, IConfiguration configuration)
   at Microsoft.AspNetCore.Hosting.BootstrapHostBuilder.RunDefaultCallbacks()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder..ctor(WebApplicationOptions options, Action`1 configureDefaults)
   at Microsoft.AspNetCore.Builder.WebApplication.CreateBuilder(String[] args)
   at Program.<Main>$(String[] args) in C:\code\project\Program.cs:line 12
```

After searching through this repo for instances of `staticwebassets` yields no results. If you have any thoughts, I'd greatly appreciate it!

**Comment by DavidVollmers** [comment]
This is fixed in version `0.6.1-preview`.

**Comment by jpsullivan** [comment]
Wonderful, thanks for the quick response and fix!
